rules/pod_status.go

```diff
@@ -0,0 +1,37 @@
+package rules
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"k8s.io/client-go/pkg/api/v1"
+)
+
+const envPodStatus = "POD_STATUSES"
+
+var _ Rule = (*podStatus)(nil)
+
+type podStatus struct {
+	reapStatuses []string
+}
+
+func (rule *podStatus) load() (bool, string, error) {
+	value, active := os.LookupEnv(envPodStatus)
+	if !active {
+		return false, "", nil
+	}
+	statuses := strings.Split(value, ",")
+	rule.reapStatuses = statuses
```

Comment:
rules.reapStatuses = strings.Split(value, “,”)

Lines 24 & 25 could be combined.

Reply:
I know I have a tendency to try to keep individual lines simple... but now I want to know how many times I've done this... it's probably a LOT